pkgs/development/python-modules/pymilter/default.nix

```
@@ -14,29 +13,22 @@
 
 buildPythonPackage rec {
   pname = "pymilter";
-  version = "1.0.5";
+  version = "1.0.6";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `build-system`